### PR TITLE
This First Commit for pushing DBT Project

### PR DESCRIPTION
**Pull request by yashdholam**
First Commit for DBT Code

Includes
2 Models : For Products Table

### CODE REVIEW

**Review by OmkarShevde — APPROVED** [review]
Looks fine

**Review by Amitjoshi7 — APPROVED** [review]
looks fine

**Review by OmkarShevde — APPROVED** [review]

File: dbt/batch2_elt_project/dbt_project.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# directory as views. These settings can be overridden in the individual model
+# files using the `{{ config(...) }}` macro.
+models:
+  batch2_elt_project:
+    # Config indicated by + and applies to all files under models/example/
+    dimension:
+      +materialized: table
+    fact:
+      +materialized: view  
+    stagung:
```
Comment (omingale0707) [inline dbt/batch2_elt_project/dbt_project.yml]: typing error

File: dbt/batch2_elt_project/models/sources.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+sources:
+  - name: bronze
+    database : dev
+    schema : bronze
+    tables:
+        - name: products
+          identifier: products
```
Comment (omingale0707) [inline dbt/batch2_elt_project/models/sources.yml]: no need of identifier if name is same

File: dbt/batch2_elt_project/dbt_project.yml (hunk 1 above)
Comment (yashdholam) [reply dbt/batch2_elt_project/dbt_project.yml]: Thank you for pointing it out . I have made the change .

File: dbt/batch2_elt_project/models/sources.yml (hunk 2 above)
Comment (yashdholam) [reply dbt/batch2_elt_project/models/sources.yml]: Yes i have changed it

**Review by omingale0707 — APPROVED** [review]

**Pull request merged**